.gitignore

```diff
@@ -141,4 +141,18 @@ cython_debug/
 # opensearch files
 test_opensearch/cover
 test_opensearch/local.py
-.ci/output
\ No newline at end of file
+.ci/output
+
+# opensearch-dsl .gitignore
```

Comment:
Lets merge the gitignore paths, there are some that are repeated in this section.

Reply:
Removed repeated lines and merged gitignore paths